<?xml version="1.0"?>
<launch>

<node respawn="false" pkg="jaco2_calibration_utils" type="record_static_data" name="record_static_data" output="screen">
    <param name="/file" type="string" value="/tmp/jaco2-2_static_torque_data1.yaml"/>
    <param name="/driver_name" type="string" value="/jaco_22_driver"/>
</node>

 <node pkg="rosbag" type="record" name="rosbag_record_diag"
       args="record -o /tmp/static_torque_calib /jaco_22_driver/out/joint_state_accs /jaco_22_driver/out/joint_angles /jaco_22_driver/out/sensor_info"/>  
</launch>
